bin/config.js

```diff
@@ -13,5 +13,9 @@ module.exports = {
         protocolModule: http,
         protocol: 'http',
         option: {}
-    }
+    },
+    appPort: 7478,
```

Comment:
if you add default prefix to these new values it would be so nice, ex: defaultAppPort: 7478

Reply:
Added prefix.